scripts/probe/probe_usage.ts

```
@@ -25,11 +35,12 @@ const main = async () => {
     console.error("OPENAI_API_KEY required");
     process.exit(1);
   }
+  console.log(`action: ${action}`);
```

ESLint

[error] Unexpected console statement.

(no-console)